bmc-state-watcher.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
```

Comment:
Please add a YADRO copyright banner with short Apache clause, see the Appendix at the bottom of the official [Apache License](https://www.apache.org/licenses/LICENSE-2.0.html).